### PR TITLE
[GEOT-5943] Expose GDAL ASRP/USRP format

### PR DESCRIPTION
**Pull request by aaime**

### CODE REVIEW

File: modules/plugin/imageio-ext-gdal/src/main/java/org/geotools/coverageio/gdal/srp/SRPFormatFactory.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+			.getLogger(SRPFormatFactory.class.getPackage().getName());
+
+	/**
+	 * Tells me if the coverage plugin to access Erdas imagine is available or
+	 * not.
+	 * 
+	 * @return <code>true</code> if the plugin is available,
+	 *         <code>false</code> otherwise.
+	 */
+	public boolean isAvailable() {
```
Comment (dromagnoli) [inline modules/plugin/imageio-ext-gdal/src/main/java/org/geotools/coverageio/gdal/srp/SRPFormatFactory.java]: this file looks like having TAB indentations

File: modules/plugin/imageio-ext-gdal/src/main/java/org/geotools/coverageio/gdal/jp2ecw/JP2ECWFormat.java [hunk 2]
```diff
@@ -21,6 +21,7 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import jdk.nashorn.internal.ir.annotations.Ignore;
```
Comment (dromagnoli) [inline modules/plugin/imageio-ext-gdal/src/main/java/org/geotools/coverageio/gdal/jp2ecw/JP2ECWFormat.java]: weird import here. I think you wanted the junit ignore instead

**Comment by aaime** [comment]
Feedback handled

**Comment by bradh** [comment]
Does this need a (follow-up?) docs update?

**Comment by aaime** [comment]
I suppose we could add one more bullet point in this page:
http://docs.geotools.org/stable/userguide/library/coverage/imageio.html
(I believe it's already quite outdated though)

**Comment by aaime** [comment]
List updated, merging

**Pull request merged**